docs/release-notes/flink-1.12.md

```diff
@@ -0,0 +1,194 @@
+
+
+#### Update internal aggregate functions to new type system [FLINK-18809](https://issues.apache.org/jira/browse/FLINK-18809)
+
+SQL queries that use the `COLLECT` function might need to be updated to the new type system. 
+
+
+### Connectors and Formats
+
+#### Remove Kafka 0.10.x and 0.11.x connectors [FLINK-19152](https://issues.apache.org/jira/browse/FLINK-19152)
+
+In Flink 1.12 we removed the Kafka 0.10.x and 0.11.x connectors. Please use the universal Kafka connector which works with any Kafka cluster version after 0.10.2.x. 
+
+Please refer to the [documentation]({% link dev/connectors/kafka.md %}) to learn about how to upgrade the Flink Kafka Connector version. 
+
+
+#### Add `flink-connector-base` to `flink-dist` [FLINK-18447](https://issues.apache.org/jira/browse/FLINK-18447)
+
+The `flink-connector-base` module is now part of `flink-dist` and is always in the classpath of the JVM in standalone deployments or docker images. 
+
+Connectors that build on top of this connector base should now declare the dependency as 'provided', similar to API dependencies. 
+
+
+#### Csv Serialization schema contains line delimiter [FLINK-19868](https://issues.apache.org/jira/browse/FLINK-19868)
+
+The `csv.line-delimiter` option has been removed from CSV format. Because the line delimiter should be defined by the connector instead of format. If users have been using this option in previous Flink version, they should alter such table to remove this option when upgrading to Flink 1.12. There should not much users using this option. 
+
+
+#### Upgrade to Kafka Schema Registry Client 5.5.0 [FLINK-18546](https://issues.apache.org/jira/browse/FLINK-18546)
+
+The `flink-avro-confluent-schema-registry` module is no longer provided as a fat-jar. You should include its dependencies in your job's fat-jar. Sql-client users can use flink-sql-avro-confluent-schema-registry fat jar. 
+
+
+#### Upgrade to Avro version 1.10.0 from 1.8.2 [FLINK-18192](https://issues.apache.org/jira/browse/FLINK-18192)
+
+The default version of Avro in flink-avro module was upgraded to 1.10. If for some reason you need an older version (you have Avro coming from Hadoop, or you use classes generated from an older Avro version), please explicitly downgrade the Avro version in your project. 
+
+NOTE: We observed a decreased performance of the Avro 1.10 version compared to 1.8.2. If you are concerned with the performance and you are fine working with an older version of Avro, consider downgrading the Avro version. 
+
+
+#### Create an uber jar when packaging flink-avro for sql client [FLINK-18802](https://issues.apache.org/jira/browse/FLINK-18802)
+
+The sql client jar was renamed to flink-sql-avro-{{site.version}}.jar, previously flink-avro-{{site.version}}-sql-jar.jar. Moreover it is no longer needed to add Avro dependencies manually. 
+
+
+### Deployment
+
+#### Enable rolling in default log settings of standalone scripts [FLINK-8357](https://issues.apache.org/jira/browse/FLINK-8357)
```

Comment:
I am afraid not only the default log settings in standalone scripts has changed. But also the JobManager and TaskManager log settings in all other deployments(e.g. Yarn, K8s, etc.).